.github/workflows/onmergerelease.yml

```
@@ -23,7 +23,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v4
         with:
-          go-version: '1.21'
+          go-version: '1.21.4'
```

you can get rid of explicit versions with:
```suggestion
          go-version-file: go.mod
          check-latest: true
```

---

updated. thank you

---

LGTM, thanks

---

In my opinion, check-latest should be removed. We do manual go version updates in each of our product releases so that it does not break due to any changes that come in with the latest changes. go-version-file change looks good.